resources/views/ignored.blade.php

```diff
@@ -3,6 +3,9 @@
 @section('content')
 <div class="bg-gray-100 font-sans relative z-0">
     <div class="max-w-6xl mx-auto pt-8">
+				<p class="mb-6 text-black-lighter">
+            Showing <span id="projectCounter">{{ $projects->count() }}</span> ignored projects and packages
```

Comment:
Nitpicky but can you snake_case this id like you did with `id="project_{{ $project->id }}"` below please?

Reply:
Sure thing